packages/components/src/components/icon/common/ic-desktop.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="48" height="43"><path fill="var(--fill-color1)" d="M0 0v32.06h48V0H0zm44.832 29.015H3.168V3.14h41.76v25.876h-.096zM5 5h38v22H5V5zm25.528 34.86h2.976V43H13.92v-3.14h3.552v-7.8h13.056v7.8z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="48" height="48"><defs><path id="a" d="M0 0h48v37.714H0z"/><path id="c" d="M0 0h21.551v3.429H0z"/><path id="e" d="M0 0h21.551v3.429H0z"/><path id="g" d="M0 48h48V0H0z"/></defs><g fill="none" fill-rule="evenodd"><mask id="b" fill="#fff"><use xlink:href="#a"/></mask><path fill="#85ACB0" d="M5.793 0h36.414C45.407 0 48 2.57 48 5.74v26.235c0 3.17-2.594 5.74-5.793 5.74H5.793c-3.2 0-5.793-2.57-5.793-5.74V5.74C0 2.57 2.594 0 5.793 0" mask="url(#b)"/><path fill="#C7E5E5" d="M6.398 3.918h34.714c1.37 0 2.48 1.117 2.48 2.494v22.441a2.486 2.486 0 01-2.48 2.494H6.398a2.486 2.486 0 01-2.48-2.494V6.412a2.487 2.487 0 012.48-2.494m8.786 40.653l1.679-6.857h10.076v6.857z"/><g transform="translate(4.898 44.571)"><mask id="d" fill="#fff"><use xlink:href="#c"/></mask><path fill="#85ACB0" d="M21.551 3.429H1.658C.742 3.429 0 2.66 0 1.714 0 .768.742 0 1.658 0H21.55v3.429z" mask="url(#d)"/></g><path fill="#C7E5E5" d="M21.061 44.571v-6.857h10.076l1.679 6.857z"/><g transform="translate(21.551 44.571)"><mask id="f" fill="#fff"><use xlink:href="#e"/></mask><path fill="#85ACB0" d="M0 0h19.893c.916 0 1.658.768 1.658 1.714 0 .947-.742 1.715-1.658 1.715H0V0z" mask="url(#f)"/></g><path fill="#E1F4F3" d="M8.96 14.204H7.347V9.077c0-.955.73-1.73 1.628-1.73H21.06V9.06H8.975l-.015 5.143z"/><mask id="h" fill="#fff"><use xlink:href="#g"/></mask><path fill="#E1F4F3" d="M23.02 8.816h1.47V7.347h-1.47z" mask="url(#h)"/></g></svg>
```

Comment:
Just checking, is this compatible with dark theme?

Reply:
@aarimm , As I checked, we only use this icon in DBot save modal and we don't have dark theme on Bot